lib/process.ml

```diff
@@ -233,7 +237,8 @@ module Blocking : S with type 'a io = 'a = struct
     | None -> Bytes.sub buf 0 (i + 1) :: acc
 
   let read_lines buf len into fd =
-    let n = Unix.read fd buf 0 len in
+    (* The EPIPE case covers an odd behavior on Windows. *)
```

Comment:
A line to the mantis issue (http://caml.inria.fr/mantis/view.php?id=7342) would be useful, in case the reader would like more details on the odd behaviour.